apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -14,40 +14,82 @@ import no.nav.etterlatte.libs.common.behandling.BehandlingType
 import no.nav.etterlatte.libs.common.behandling.Persongalleri
 import no.nav.etterlatte.libs.common.behandling.RevurderingAarsak
 import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+import no.nav.etterlatte.libs.common.grunnlag.NyeSaksopplysninger
+import no.nav.etterlatte.libs.common.grunnlag.opplysningstyper.Opplysningstype
+import no.nav.etterlatte.libs.common.grunnlag.opplysningstyper.SoeknadMottattDato
+import no.nav.etterlatte.libs.common.gyldigSoeknad.GyldighetsResultat
+import no.nav.etterlatte.libs.common.gyldigSoeknad.VurderingsResultat
 import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
 import no.nav.etterlatte.libs.common.sak.Sak
+import no.nav.etterlatte.libs.common.tidspunkt.Tidspunkt
+import no.nav.etterlatte.libs.common.tidspunkt.toLocalDatetimeUTC
+import no.nav.etterlatte.libs.common.tidspunkt.toTidspunkt
+import no.nav.etterlatte.libs.common.toJsonNode
 import no.nav.etterlatte.oppgaveny.OppgaveServiceNy
-import no.nav.etterlatte.sak.SakDao
-import no.nav.etterlatte.sak.filterSakerForEnheter
+import no.nav.etterlatte.sak.SakService
 import org.slf4j.LoggerFactory
 import java.time.LocalDateTime
 
 class BehandlingFactory(
     private val oppgaveService: OppgaveServiceNy,
     private val grunnlagService: GrunnlagService,
     private val revurderingService: RevurderingService,
-    private val sakDao: SakDao,
+    private val gyldighetsproevingService: GyldighetsproevingService,
+    private val sakService: SakService,
     private val behandlingDao: BehandlingDao,
     private val hendelseDao: HendelseDao,
     private val behandlingHendelser: BehandlingHendelserKafkaProducer,
     private val featureToggleService: FeatureToggleService
 ) {
     private val logger = LoggerFactory.getLogger(this::class.java)
+
+    fun opprettBehandling(behov: NyBehandlingRequest): Behandling {
```

Comment:
Synes det er rart at denne heter `opprettBehandling` samtidig som den kaller `opprettBehandling` i seg som er annen metode her. Den gjør vel strengt talt mer enn å opprette en behandling og. Hva med `opprettManuellBehandlingMedSak` ellernoe.

Reply:
Synes vi kan ha en test på denne nye flyten for opprettelse av behandling "synk" også.